tangy-form-item.js

```diff
@@ -466,18 +466,18 @@ export class TangyFormItem extends PolymerElement {
   onOpenChange(open) {
     // Close it.
     if (open === false) {
-      this.$.content.innerHTML = ''
+      this.innerHTML = ''
     }
     // Open it, but only if empty because we might be stuck.
-    if (open === true && this.$.content.innerHTML === '') {
+    if (open === true && this.innerHTML === '') {
       this.openWithContent(this.template)
     }
     
   }
 
   openWithContent(contentHTML) {
-    this.$.content.innerHTML = contentHTML
-    this.$.content
+    this.innerHTML = contentHTML
```

Comment:
This line and the addition of a `<slot>` element in the template is the big change. Everything else is simply adjusting for the fact that the inputs will no longer be found in the shadow DOM.